AppCenterIdentity/AppCenterIdentity/MSIdentity.h

```diff
@@ -1,7 +1,35 @@
 #import "MSServiceAbstract.h"
+#import "MSUserInformation.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * Completion handler triggered when sign-in completed.
+ *
+ * @param userInformation User information for signed in user.
+ * @param error Error for sign-in failure.
+ */
+typedef void (^MSSignInCompletionHandler)(MSUserInformation *_Nullable userInformation, NSError *_Nullable error);
+
+/**
+ * Error code for Identity.
+ */
+typedef NS_ENUM(NSInteger, MSIdentityErrorCode) {
+  MSIdentityErrorServiceDisabled = -420000,
+
+  MSIdentityErrorPreviousSignInRequestNotCompleted = -420001
```

Comment:
maybe `MSIdentityErrorAnotherSignInInProgress` is more clear about the purpose? 😄

Reply:
Are you sure? The double In does not look better.

Reply:
I agree that Di's proposal is clearer about the intention.
How about `MSIdentityErrorAnotherSignInIsInProgress`? It's already long and `InIsIn` reads a little less weird than `InIn`?
My favorite would be `MSIdentityErrorPreviousSignInRequestInProgress`?

Reply:
I prefer the latter